controller/persistence/session_store_test.go

```diff
@@ -234,7 +234,7 @@ func (ctx *TestContext) testUpdateSessions(_ *testing.T) {
 	earlier := time.Now()
 	time.Sleep(time.Millisecond * 50)
 
-	mutateCtx := change.New().SetSource("test").NewMutateContext()
+	mutateCtx := change.New().NewMutateContext()
```

Comment:
I don't seem to follow from this alone why "test" is sometimes desirable and others not. Can you explain?

Reply:
I just gave up half way through and started ripping them back out. I can take out the rest, if you prefer.